cli/command/formatter/stats.go

```diff
@@ -145,8 +145,10 @@ func (c *containerStatsContext) Container() string {
 
 func (c *containerStatsContext) Name() string {
 	c.AddHeader(nameHeader)
-	name := c.s.Name[1:]
-	return name
+	if len(c.s.Name) > 1 {
+		return c.s.Name[1:]
+	}
+	return ""
```

Comment:
Makes sense, but for consistency (see CPUPerc(), MemUsage(), ...), I thought `"--"` seems better.
But this is nitpicky and overall LGTM.